hc/front/tests/test_add_pushbover.py

```diff
@@ -38,3 +39,15 @@ def test_it_validates_nonce(self):
         assert r.status_code == 403
 
     ### Test that pushover validates priority
+    def test_validate_priority(self):
```

Comment:
@georgreen @davidmukiibi Could either of you explain what is going on in this test method.